acct-user/sddm/sddm-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for sddm"
+ACCT_USER_ID=219
+ACCT_USER_GROUPS=( sddm )
```

Comment:
Should be ```video``` added as secondary group? Current ebuilds do that.

Reply:
yes. only because of nvidia.

Reply:
I did look into alternatives, but at this point I can safely say this requirement is not going away in a foreseeable future, so yes indeed.